### PR TITLE
Wells USD values

### PR DESCRIPTION
**Pull request by cujowolf**
Update handlers to calculate and save well amounts in USD. Current limitation on implementation is an expectation of a two token well.

### CODE REVIEW

File: projects/subgraph-wells/subgraph.yaml [hunk 1]
```diff
@@ -37,6 +39,10 @@ templates:
       abis:
         - name: Well
           file: ./abis/Well.json
+        - name: CurvePrice
+          file: ./abis/CurvePrice.json
+        - name: ERC20
+          file: ./abis/ERC20.json
       eventHandlers:
         - event: AddLiquidity(uint256[],uint256)
           handler: handleAddDeposit
```
Comment (marshall-bean) [inline projects/subgraph-wells/subgraph.yaml:48]: Shouldn't this be `handleAddLiquidity` ?

**Review by marshall-bean — CHANGES_REQUESTED** [review]
Some observations about the data in the subgraph:
- lastPriceUSD showing as 0 for all tokens, leads all USD values to be 0 in the subgraph. Expected?
- swaps don't seem to be getting indexed, not yet sure why

File: projects/subgraph-wells/subgraph.yaml (hunk 1 above)
Comment (cujowolf) [reply projects/subgraph-wells/subgraph.yaml]: Updated

**Review by marshall-bean — APPROVED** [review]

File: projects/subgraph-wells/src/WellHandler.ts [hunk 2]
```diff
@@ -3,88 +3,95 @@ import { loadOrCreateAccount } from "./utils/Account";
...
+  incrementWellSwap,
+  incrementWellWithdraw,
+  loadWell,
+  updateWellLiquidityTokenBalance,
+  updateWellTokenBalances,
+  updateWellVolumes
+} from "./utils/Well";
 
 export function handleAddLiquidity(event: AddLiquidity): void {
+  loadOrCreateAccount(event.transaction.from);
```
Comment (marshall-bean) [inline projects/subgraph-wells/src/WellHandler.ts:18]: withdraws and deposits showing 0 for `amountUSD` always (not related to this line just mentioning it via PR review)

**Review by marshall-bean — CHANGES_REQUESTED** [review]

**Review by marshall-bean — APPROVED** [review]

**Pull request merged**